src/deprecated-layers/choropleth-layer/choropleth-layer-vertex.glsl.js

```diff
@@ -26,7 +26,6 @@ attribute vec4 colors;
 attribute vec3 pickingColors;
 
 uniform float opacity;
-uniform float renderPickingBuffer;
```

Comment:
These are deprecated layers, we should change them as little as possible. But I assume this was simply not used here.